script/zonemaster-cli

```diff
@@ -114,14 +114,50 @@ Print the effective profile used in JSON format, then exit.
 =item B<--test>=TESTCASE, B<--test>=TESTMODULE
 
 Limit the testing suite to run only the specified tests.
-Can be specified multiple times.
-(default: all test cases)
-
-=for :man This can be the name of a testing module, in which case all test cases from
-that module will be run, or the name of a module followed by a slash and the
-name of a test case (test case identifier) in that module, or the name of the
-test case.
-This option is case-insensitive.
+(default: the empty string)
+
+=begin :man
+
+EXPR is an expression specifying or modifying the set of tests to run.
+Here are some examples:
+
+=over 4
+
+=item --test=B<nameserver>
+
+runs the entire Nameserver test module, and nothing else.
```

Comment:
```suggestion
runs the entire Nameserver test module, including any test cases
disabled by the profile, but nothing else.
```

Reply:
I agree that the documentation should make this clear, but I don't think it needs to be repeated for every example. I'll add a separate paragraph clarifying how it interacts with other options.